parlai/crowdsourcing/tasks/model_chat/impl.py

```diff
@@ -47,7 +50,17 @@ def run_task(cfg: DictConfig, task_directory: str, world_module=None):
     soft_block_mturk_workers(cfg=cfg, db=db, soft_block_qual_name=soft_block_qual_name)
 
     # Init
-    shared_state = SharedModelChatTaskState(world_module=world_module)
+    if cfg.mephisto.blueprint.allowed_worker_qualification is not None:
```

Comment:
Should we add a TODO to bubble this functionality up to all ParlAI Mephisto tasks in the future?